controllers/main.go

```diff
@@ -1210,16 +1214,24 @@ func (controller *MainController) Index(c web.C, r *http.Request) (string, int)
 	t := controller.GetTemplate(c)
 
 	// execute the named template with data in c.Env
-	widgets := helpers.Parse(t, "home", c.Env)
-
+	widgets, err := helpers.Parse(t, "home", c.Env)
+	if err != nil {
+		log.Infof("helpers.Parse: home failed: %v", err)
+		return "/error?r=/stats", http.StatusSeeOther
+	}
```

Comment:
This changes behavior, but it looks reasonable.

Reply:
Agreed, tested this out and the new behaviour is an improvement.